integration_tests/commands/http/copy_test.go

```diff
@@ -85,7 +85,7 @@ func TestCopy(t *testing.T) {
 				{Command: "TTL", Body: map[string]interface{}{"key": "k1"}},
 				{Command: "TTL", Body: map[string]interface{}{"key": "k2"}},
 			},
-			expected: []interface{}{"OK", float64(1), float64(-1), float64(-1)},
```

Comment:
The data type of these results should not change with this migration. Same for all other instances.

Reply:
Since the functions are returning `clientio.IntegerOne`, `clientio.IntegerZero`, etc., the decoder in `(e *HTTPCommandExecutor) FireCommand` is decoding it to string `"1"`, `"0"`, etc. Therefore I had to change the expected values from float to string where the migrated functions have been used.

Reply:
@lucifercr07 could you please confirm this behavior?

Reply:
Hey @JyotinderSingh, @SyedMa3, @lucifercr07, I think this is resolved. Please confirm.

Reply:
Will look at it tonight, and will update the PR

Reply:
Yes has been resolved as part of #1204